internal/renderers/skia/lib.rs

```diff
@@ -88,10 +89,29 @@ impl SkiaRenderer {
             path_cache: Default::default(),
             rendering_metrics_collector: Default::default(),
             rendering_first_time: Cell::new(true),
-            surface: Box::new(surface),
+            surface: RefCell::new(Some(Box::new(surface))),
         }
     }
 
+    /// Reset the surface to a new surface. (destroy the previously set surface if any)
+    pub fn set_surface(&self, surface: impl Surface + 'static) {
```

Comment:
Regarding the API, is the Surface trait public?  or should we just have the other function public. 
Should the new* functions  be removed?